scrapers/Xevunleashed.yml

```diff
@@ -0,0 +1,34 @@
+name: "XevUnleashed"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - xevunleashed.com/updates/
+    scraper: sceneScraper
+
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //span[@class="update_title"]/text()
+      Date:
+        selector: //span[@class="availdate"]/text()
+        concat: " "
+        postProcess:
+          - replace:
+              - regex: (\d{2}\/\d{2}\/\d{4}).*
+                with: $1
+          - parseDate: 01/02/2006
+      Details: //meta[@name="description"]/@content
+      Studio:
+        Name:
+          fixed: XevUnleashed
+      Image: //meta[@name="twitter:image"]/@content
+      Tags:
+        Name: //span[@class="update_tags"]//a
+      Performers:
+        Name:
+          selector: //meta[@name="keywords"]/@content
+          split: ','
```

Comment:
Where there scenes where the performers are mentioned in the keywords? Even `Xev Bellringer` isnt mentioned in any of the scenes i tried

Reply:
It gets them when there are 2 performers (so like... less than 10 scenes overall I think)

Not sure if it's worth it...

Reply:
Maybe add Xev Bellringer as a fixed name performer?

Reply:
Can you share a couple of links to scenes where this didn't work ? I tried to find some but couldn't find any where her name isn't in the keywords

Reply:
Hm... i think it was the few scenes i tested.
I tested mostly scenes from here to see if any extra performers were in the keywords... <https://xevunleashed.com/categories/mnage--trois.html>

Reply:
@JackDawson94 i thinks its ok to merge as is ? Is it ok to merge now?

Reply:
Ok to merge yeah